.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,9 @@
+
+
+## ARM API Information (Control Plane)
+<details>
```

Comment:
Do we really want to these to be collapsed sections? I worry a little that folks will not ever expand them and read or check any of the options.

Reply:
Yeah, I see that point. @markweitzel any thoughts?

Reply:
The collapsible sections do have the advantage of focusing people's attention on ARM vs Data plane until we are able to have 2 templates.